orders/models.py

```diff
@@ -71,3 +73,38 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    @staticmethod
+    def get_capacity_of_daycare_in_dates_range(daycare_id, start_date, end_date):
+        relevant_orders = Order.objects.filter(daycare_id=daycare_id, status__in=['A', 'O'])
+        start_date = datetime.date(year=start_date.year, month=start_date.month, day=start_date.day)
```

Comment:
why do we declare start_date again? since we don't use python's `Typing` it is a bit confusing. maybe formatted_start_date will be better.

Reply:
Sorry if it wasn't clear that this function took from #95, I guess you already mention this comment there.